first.cpp

```diff
@@ -26,6 +26,11 @@ class Foo
     {
         std::cout << "Printing stuff" << std::endl;
     }
+
+    void greatstuff()
+    {
+        std::cout << "here you go..." << std::endl;
```

Comment:
All right, how about this though?
if (everything_is_all_right)
   std::cout << "here you go..." << std::endl;